whois-nrtm4/src/main/java/net/ripe/db/nrtm4/dao/UpdateNrtmFileRepository.java

```diff
@@ -29,34 +29,35 @@
 import java.util.Map;
 
 @Repository
-public class NrtmFileRepository {
+public class UpdateNrtmFileRepository {
 
-    private static final Logger LOGGER = LoggerFactory.getLogger(NrtmFileRepository.class);
+    private static final Logger LOGGER = LoggerFactory.getLogger(UpdateNrtmFileRepository.class);
     private final JdbcTemplate jdbcTemplate;
     private final DateTimeProvider dateTimeProvider;
     private final NamedParameterJdbcTemplate namedParameterJdbcTemplate;
 
-    public NrtmFileRepository(@Qualifier("nrtmDataSource") final DataSource dataSource, final DateTimeProvider dateTimeProvider) {
+
+    public UpdateNrtmFileRepository(@Qualifier("nrtmMasterDataSource") final DataSource dataSource, final DateTimeProvider dateTimeProvider) {
         this.jdbcTemplate = new JdbcTemplate(dataSource);
         this.dateTimeProvider = dateTimeProvider;
         this.namedParameterJdbcTemplate = new NamedParameterJdbcTemplate(jdbcTemplate);
     }
 
-    @Transactional
+    @Transactional("nrtmTransactionManager")
     public void saveDeltaVersion(final NrtmVersionInfo version, final int serialIDTo, final List<DeltaFileRecord> deltas) throws JsonProcessingException {
-       if(deltas.isEmpty()) {
-           LOGGER.info("No delta changes found for source {}", version.source().getName());
-           return;
-       }
+        if (deltas.isEmpty()) {
+            LOGGER.info("No delta changes found for source {}", version.source().getName());
+            return;
+        }
 
```

Comment:
Why there is no transactional annotation on delete?

Reply:
oh my fault, it should have that annotation as well. I will add it, thanks!